### PR TITLE
add missing attributes for per unit view

### PR DESCRIPTION
**Pull request by EtienneLt**
**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

**What kind of change does this PR introduce?**

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [ ] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**

**Other information**:

### CODE REVIEW

**Review by geofjamg — COMMENTED** [review]
Did you check that all attributes that are now accessible are correctly per-united? For example target_v for transformer 2 and 3, shunt ? Also all target p and q (hvdc lines etc ) ?

**Comment by EtienneLt** [comment]
> Did you check that all attributes that are now accessible are correctly per-united? For example target_v for transformer 2 and 3, shunt ? Also all target p and q (hvdc lines etc ) ?

I will add some tests

**Comment by EtienneLt** [comment]
> Did you check that all attributes that are now accessible are correctly per-united? For example target_v for transformer 2 and 3, shunt ? Also all target p and q (hvdc lines etc ) ?

well all targetv for transformer and target p ans q for hvdc are default attributes so they were already perunit 
I added the non default attributes for generators and vsc stations (max_q_at_p, ....)

**Comment by EtienneLt** [comment]
update_ratio_tap_changers is missing

**Review by geofjamg — APPROVED** [review]

File: pypowsybl/network/impl/perunit.py [hunk 1]
```diff
@@ -519,6 +528,20 @@ def update_lcc_converter_station(self, df: pd.DataFrame = None, **kwargs: ArrayL
...
 
+    def update_ratio_tap_changers(self, df: pd.DataFrame = None, **kwargs: ArrayLike) -> None:
+        to_update = _adapt_to_dataframe(ElementType.RATIO_TAP_CHANGER, df, **kwargs).copy()
+        ref = self._network.get_ratio_tap_changers(all_attributes=True)
+        voltage_levels = ref[[]].merge(
+            self.get_2_windings_transformers()[['voltage_level1_id', 'voltage_level2_id']],
+            left_index=True, right_index=True)
+        nominal_v1 = self._get_indexed_nominal_v(voltage_levels, 'voltage_level1_id')
+        nominal_v2 = self._get_indexed_nominal_v(voltage_levels, 'voltage_level2_id')
+        regulated_side_nominal_voltage = nominal_v2
```
Comment (geofjamg) [inline pypowsybl/network/impl/perunit.py:539]: Why do we get a different peruniting depending on if we have the target_v inside the dataframe?

**Review by geofjamg — CHANGES_REQUESTED** [review]

**Pull request closed without merging**